mlx/traceability/directives/item_directive.py

```diff
@@ -63,7 +63,8 @@ def _process_relationships(self, *args):
         All targets get naturally sorted per relationship.
         """
         for rel, targets in self._item.all_relations:
-            self._list_targets_for_relation(rel, targets, *args)
+            if rel not in self['hidetype']:
+                self._list_targets_for_relation(rel, targets, *args)
```

Comment:
Do you want a warning logged when a relation type in `self['hidetype']` is not in `self._item.all_relations`? 

Keep in mind that it is possible to conditionally change/add relations with the `only`-directive and `item-relink`/`item-link` directive. And you can't really use the `:hidetype:` conditionally.

Reply:
checking against `self._item.all_relations()` will not work for my use case, as the relation-type might not be there for this particular item. We just want, if it's there don't show it...
I added a check for the hidetype being present in `TraceableCollection::relations`, which we also do for other places where relation-types are given (item-tree, item-matrix, ...).